configscanning/githubrepo.py

```diff
@@ -299,7 +299,7 @@ def create_tag(self, name, message):
         self.repo.create_tag(
             name,
             self.repo.head.target,
-            pygit2.GIT_REF_OID,  # Is this correct? It's not documented what's allowed here.
+            True,
```

Comment:
Is this definitely correct? It seems to be documented as requiring an enums.ObjectType instead of Boolean value here.

Reply:
I've updated it. The repository seems to have changed a few times